lib/artsy-eventservice/artsy/event_service.rb

```diff
@@ -2,6 +2,23 @@
 # frozen_string_literal: true
 module Artsy
   module EventService
+
+    # Post data without an event, must be serialized.
+    def self.post_data(topic:, data:, routing_key:)
+      RabbitMQConnection.get_channel do |channel|
```

Comment:
Nice! I'd suggest move this part of code to `Publisher` [here](https://github.com/artsy/artsy-eventservice/blob/master/lib/artsy-eventservice/artsy/event_service/publisher.rb#L4) as a generic method and update here and `post_event` in [`Publisher`](https://github.com/artsy/artsy-eventservice/blob/master/lib/artsy-eventservice/artsy/event_service/publisher.rb#L9) to use that generic method.

Reply:
I agree that this should ideally be used by the other location rather than duplicated there. And if this is an addition to the gem's public API, it should have a spec.